k8s/charts/rtsp-camera-to-pravega/values.yaml

```diff
@@ -11,6 +11,7 @@ appParameters:
   CAMERA_PROTOCOLS: "tcp"
   GST_DEBUG: "WARNING,rtspsrc:INFO,rtpbin:INFO,rtpsession:INFO,rtpjitterbuffer:INFO,h264parse:WARN,pravegasink:DEBUG"
   pravega_client_tls_cert_path: "/etc/ssl/certs/ca-certificates.crt"
+  PYTHONUNBUFFERED: "1"
```

Comment:
Although this works, such a change would also need to be made to the K8s operator, docker scripts, etc. I think we can just edit the first line of the .py file as shown in https://stackoverflow.com/a/21433197/5890553.
```
#!/usr/bin/env -S python -u
```

Reply:
done